test-src/org/pentaho/reporting/platform/plugin/async/PentahoAsyncExecutionTest.java

```diff
@@ -117,7 +120,7 @@ public void testListenerFailExecution() throws Exception {
 
     PentahoAsyncReportExecution exec = createMockCallable();
     // simulates queuing
-    exec.notifyTaskQueued( UUID.randomUUID() );
+    exec.notifyTaskQueued( UUID.randomUUID(), null );
```

Comment:
This call should actually error out. If you have a list parameter, never pass in Null, demand that your API user passes in Collections.emptyList() instead.